dev-python/jupyter_client/jupyter_client-5.2.2.ebuild

```diff
@@ -0,0 +1,39 @@
+# Copyright 1999-2018 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+
+PYTHON_COMPAT=( python2_7 python3_{4,5,6} )
+PYTHON_REQ_USE="threads(+)"
+
+inherit distutils-r1
+
+DESCRIPTION="Jupyter protocol implementation and client libraries"
+HOMEPAGE="http://jupyter.org"
+SRC_URI="mirror://pypi/${PN:0:1}/${PN}/${P}.tar.gz"
+
+LICENSE="BSD"
+SLOT="0"
+KEYWORDS="~amd64 ~arm64 ~x86"
+IUSE="test"
+
+RDEPEND="
+	dev-python/python-dateutil[${PYTHON_USEDEP}]
+	dev-python/traitlets[${PYTHON_USEDEP}]
+	dev-python/jupyter_core[${PYTHON_USEDEP}]
+	>=dev-python/pyzmq-14.4.0[${PYTHON_USEDEP}]
+	"
+DEPEND="${RDEPEND}
+	dev-python/entrypoints[${PYTHON_USEDEP}]
+	dev-python/setuptools[${PYTHON_USEDEP}]
```

Comment:
This is RDEP because of entry_points in setup.py. Also, I'm not sure what 'entrypoints' thing but it also looks like a potential RDEP.

Reply:
I somehow don't get why it should be a runtime dependency. With 'entry_points ', some console_scripts are created at installation, but that's it. Please correct my if I am wrong.

Reply:
Uninstall it and see for yourself ;-).

Reply:
Well that is currently not an option, so I'll just follow your advice ;-)